docker-images/artifacts/kafka-thirdparty-libs/cc/pom.xml

```diff
@@ -16,7 +16,7 @@
     </licenses>
 
     <properties>
-        <cruise-control.version>2.5.82</cruise-control.version>
+        <cruise-control.version>2.5.85</cruise-control.version>
```

Comment:
The new test for the nexted YAMLs is great. But could you please leave the Cruise Control update for a separate PR?

Reply:
Reverted it.

Reply:
Just to be clear - if you want to update Cruise Control it is fine -> just use separate PR ;-)